<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourDialog</class>
 <widget class="QDialog" name="ContourDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="isEdgeDetectcheckBox">
      <property name="text">
       <string>Edge Detect Before Drawing</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="drawPushButton">
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Edge Threshold</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="closePushButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Contour Type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="edgeThreshspinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="contourTypeCombBox">
      <item>
       <property name="text">
        <string>select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>External</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>List</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hierarchy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Two Level Hierarchy</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
